script/app.js

```diff
@@ -139,3 +147,77 @@ const completeTodo = (taskId) => {
     todos = updatedTodos;
     renderTodos();
 };
+
+const editTodo = (taskId) => {
+    const task = todos.find((task) => task.id === taskId);
+    editingId = task.id;
```

Comment:
aren't we getting the ID from taskId?

Reply:
Yes, we got it, but in edit mode, our delete button has two functionalities: one to cancel the edit and go back to a normal state, and another to delete the task. So I need an editingId=task.id as a global declaration. Otherwise, the condition is not working properly.

Reply:
if in that case, we don't need to use an id, we can take a boolean for toggling, or we can just toggle using classname